crates/bevy_window/src/window.rs

```diff
@@ -619,20 +621,31 @@ pub struct WindowDescriptor {
     /// The position on the screen that the window will be centered at.
     /// If set to `None`, some platform-specific position will be chosen.
     pub position: Option<Vec2>,
+    /// Sets minimum and maximum resize limits.
     pub resize_constraints: WindowResizeConstraints,
-    /// Overrides the [`Window`](crate::Window)'s ratio of physical pixels to logical pixels.
+    /// Overrides the window's ratio of physical pixels to logical pixels.
     pub scale_factor_override: Option<f64>,
+    /// Sets the title that displays on the window top bar, on the system task bar and other OS specific places.
+    /// ## Platform-specific
+    /// - Web: Unsupported.
     pub title: String,
+    /// Controls when is a frame presented to the screen.
```

Comment:
```suggestion
    /// Controls when a frame is presented to the screen.
```

This wording is easier to understand.